docs/pull_request_guidelines.md

```diff
@@ -0,0 +1,36 @@
+## Creating PR
```

Comment:
I agree that the ordered lists don't look as nice format-wise, but I did it that way so that we can point to a specific location in the document more easily when commenting on PRs. For example, if someone has uppercase letters in their branch name you can point them to 1.i in "Creating a PR".

Reply:
fair enough :)